code/modules/europa/clothing/under.dm

```diff
@@ -55,6 +55,36 @@
 	desc = "A pair of shorts cut off just above the knee. For when the air gets a bit stuffy."
 	icon_state = "shorts"
 
+/obj/item/clothing/under/europa/skirts
+	name = "gray skirt"
+	desc = "A casual grey skirt for around the office."
+	icon_state = "skirt-gray"
+
+/obj/item/clothing/under/europa/skirts/blue
+	name = "blue skirt"
+	desc = "A casual navy blue skirt for around the office."
+	icon_state = "skirt-blue"
+
+/obj/item/clothing/under/europa/skirts/black
+	name = "black skirt"
+	desc = "A casual black skirt for around the office."
+	icon_state = "skirt-black"
+
+/obj/item/clothing/under/europa/skirts/skater
+	name = "red skater skirt"
+	desc = "Skates not include,"
```

Comment:
That is a mistake. I'll fix it when I am at a PC.

On Tuesday, January 5, 2016, Zuhayr <notifications@github.com> wrote:

> In code/modules/europa/clothing/under.dm
> <https://github.com/Yonaguni/EuropaStation/pull/302#discussion_r48929949>:
>
> > +	desc = "A casual grey skirt for around the office."
> > +	icon_state = "skirt-gray"
> > +
> > +/obj/item/clothing/under/europa/skirts/blue
> > +	name = "blue skirt"
> > +	desc = "A casual navy blue skirt for around the office."
> > +	icon_state = "skirt-blue"
> > +
> > +/obj/item/clothing/under/europa/skirts/black
> > +	name = "black skirt"
> > +	desc = "A casual black skirt for around the office."
> > +	icon_state = "skirt-black"
> > +
> > +/obj/item/clothing/under/europa/skirts/skater
> > +	name = "red skater skirt"
> > +	desc = "Skates not include,"
>
> include,
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Yonaguni/EuropaStation/pull/302/files#r48929949>.
>